tests/integration/test_fim/test_registry/test_registry_report_changes/test_disk_quota/test_registry_disk_quota_values.py

```diff
@@ -176,34 +177,29 @@ def test_disk_quota_values(key, subkey, arch, value_name, tags_to_apply, size,
 
     tags:
         - scheduled
-        - time_travel
-    '''
-    check_apply_test(tags_to_apply, get_configuration['tags'])
-    value_content = generate_string(size, '0')
-    values = {value_name: value_content}
+    """
+    values = create_values_content(value_name, size)
 
     _, diff_file = calculate_registry_diff_paths(key, subkey, arch, value_name)
 
     def report_changes_validator_no_diff(event):
         """Validate content_changes attribute exists in the event"""
-        assert event['data'].get('content_changes') is None, 'content_changes isn\'t empty'
+        assert event["data"].get("content_changes") is None, ERR_MSG_CONTENT_CHANGES_NOT_EMPTY
 
     def report_changes_validator_diff(event):
         """Validate content_changes attribute exists in the event"""
-        assert os.path.exists(diff_file), '{diff_file} does not exist'
-        assert event['data'].get('content_changes') is not None, 'content_changes is empty'
+        assert os.path.exists(diff_file), "{diff_file} does not exist"
+        assert event["data"].get("content_changes") is not None, ERR_MSG_CONTENT_CHANGES_EMPTY
```

Comment:
This method is the same on 2 or 3 tests. Could you create this method on a common path to not duplicate code, please?

Reply:
The method needs access to the diff variable and then needs to be passed by reference to the to registry_value_update. When the method is passed by reference I cannot feed it the diff_file as parameter there, so it needs to be defined inside the tests.